webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -33,20 +96,35 @@ export const ModuleStreamsTab = () => {
     __('Installation status'),
     __('Installed profile'),
   ];
+  const COLUMNS_TO_SORT_PARAMS = {
+    [columnHeaders[0]]: 'name',
+    [columnHeaders[1]]: 'status',
+    [columnHeaders[3]]: 'installed_profiles',
+  };
+
+  const {
+    pfSortParams, apiSortParams,
+    activeSortColumn, activeSortDirection,
+  } = useTableSort({
+    allColumns: columnHeaders,
+    columnsToSortParams: COLUMNS_TO_SORT_PARAMS,
+    initialSortColumnName: 'Name',
+  });
 
   const fetchItems = useCallback(
     (params) => {
       if (!hostId) return hostIdNotReady;
       return getHostModuleStreams(
         hostId,
-        params,
+        { ...apiSortParams, ...params },
```

Comment:
are you sure that you don't want to do `  { ...params, ...apiSortParams },` here? 
In general I think you'd want to overwrite any of the previous params no?